frontend/src/components/Modals/Question.vue

```diff
@@ -56,12 +56,14 @@
 						type="select"
 						:options="['Choices', 'User Input', 'Open Ended']"
 						class="pb-2"
+						:required="true"
 					/>
 					<div v-if="question.type == 'Choices'" class="divide-y border-t">
 						<div v-for="n in 4" class="space-y-4 py-2">
 							<FormControl
 								:label="__('Option') + ' ' + n"
 								v-model="question[`option_${n}`]"
+								:required="n == 1 ? true : false"
```

Comment:
minimum 2 options are required for Choices

Reply:
Hey @pateljannat Please check the video—only one option is required. 🤔 ??

https://github.com/user-attachments/assets/6b677a99-dc9d-4cca-b828-51a64ca0fcdb

Reply:
Update: The form has been saved without options. Is this an issue? If so, should I fix it in this PR?

https://github.com/user-attachments/assets/24c29d89-fda7-4eab-85e9-cc3d256500a2

Reply:
> Hey @pateljannat Please check the video—only one option is required. 🤔 ??

If you check the LMS Question doctype, option_2 field has a mandatory_depends_on condition. This was getting triggered properly from the desk, but not working for the portal or when the question was created through API. 

Added a validation to fix the same https://github.com/frappe/lms/pull/1133

Please add required icon for option_2 if type is Choices.

Reply:
Done
![image](https://github.com/user-attachments/assets/15abcf16-9d12-4418-bb26-ce251b37bc12)